go/vt/topo/etcd2topo/client.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2014, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package etcd2topo
+
+import (
+	"log"
+	"time"
+
+	"github.com/coreos/etcd/client"
+)
+
+func newEtcdClient(machines []string) (client.KeysAPI, error) {
+	cfg := client.Config{
+		Endpoints:               machines,
+		Transport:               client.DefaultTransport,
+		HeaderTimeoutPerRequest: 10 * time.Second,
+	}
+	c, err := client.New(cfg)
+	if err != nil {
+		log.Fatal(err)
+	}
+	kapi := client.NewKeysAPI(c)
```

Comment:
With the etcdv2 API, we used to set the default to quorum reads. This was necessary to avoid race conditions because all our code assumes topo reads are consistent.

I'm not sure if there's a way to do that for the new client API. If not, you'll have to make sure to set `Quorum: true` for every call to `Get()`:

https://godoc.org/github.com/coreos/etcd/client#GetOptions